package.json

```diff
@@ -64,6 +64,7 @@
     "@types/cookie": "^0.3.3",
     "chalk": "^4.0.0",
     "cookie": "^0.4.1",
+    "cross-fetch": "^3.0.5",
```

Comment:
Can we try to reuse `node-fetch`, which we already use?

Conditionally deciding on the `fetch` would be enough (their call signatures should be identical):

```ts
// src/context/fetch.ts
import { isNodeProcess } from '../utils/isNodeProcess'

const useFetch = isNodeProcess() ? require('node-fetch') : fetch
```

The goal here is to conditionally require the installed dependency, only when used in a NodeJS environment. It may require to properly reference `fetch`, but we can handle that.